1-exercises/B-array-some/exercise.js

```diff
@@ -16,9 +16,13 @@ let students = ["Islam", "Lesley", "Harun", "Rukmini"];
 let mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
 let pairs = pairsByIndex.map(function (indexes) {
-  let student = students[indexes[0]];
-  let mentor = mentors[indexes[1]];
-  return [student, mentor];
+  if (indexes !== null) {
```

Comment:
Your condition works fine, so well done for that! but the aim of this exercise is using some() method on pairsByIndex array. Could you think of using some() instead of line 19.